components/alert_banner.scss

```diff
@@ -0,0 +1,83 @@
+.AlertBanner {
+    border-radius: 4px;
+    border: 1px solid;
+    padding: 16px;
+    display: flex;
+    overflow: hidden;
+
+    &.danger {
+        background: rgba(var(--sys-error-text-color-rgb), 0.08);
+        border-color: rgba(var(--sys-error-text-color-rgb), 0.24);
+
+        .AlertBanner__icon i {
+            color: var(--sys-error-text);
+        }
+    }
+
+    &.info {
+        background: rgba(var(--sys-sidebar-text-active-border-rgb), 0.08);
+        border-color: rgba(var(--sys-sidebar-text-active-border-rgb), 0.24);
+
+        .AlertBanner__icon i {
+            color: var(--sys-sidebar-text-active-border);
+        }
+    }
+
+    &.warning {
+        background: rgba(var(--sys-away-indicator-rgb), 0.08);
+        border-color: rgba(var(--sys-away-indicator-rgb), 0.24);
+
+        .AlertBanner__icon i {
+            color: var(--sys-away-indicator);
+        }
+    }
+}
+
+.AlertBanner__icon {
+    flex-grow: 0;
+    flex-shrink: 0;
+    font-size: 16px;
+    line-height: 20px;
+}
+
+.AlertBanner__body {
+    flex-grow: 1;
+    flex-shrink: 1;
+    margin-left: 12px;
+    font-size: 14px;
+    line-height: 20px;
+    color: var(--sys-center-channel-color);
+}
+
+.AlertBanner__title {
+    font-weight: 600;    
+}
+
+.AlertBanner__message {
+    margin-top: 8px;
+}
+
+.AlertBanner__closeButton {
+    background: none;
+    border: none;
+    position: relative;
+    padding: 0;
+    height: 24px;
+    width: 24px;
+    border-radius: 4px;
+    top: -8px;
+    right: -8px;
+    
+    i {
+        color: rgba(var(--sys-center-channel-color-rgb), 0.56);
+        font-size: 16px;
+
+        &::before {
+            margin: 0;
+        }
+    }
+
+    &:hover {
+        background: rgba(0, 0, 0, 0.08);
```

Comment:
Probably want a var here for themes

Reply:
It's just a dark overlay, this is actually something I meant to bring with @matthewbirtch actually.
There's no hover state specified for these panels, so I added this 8% darken overlay, let me know if that's okay.